engines/made/made.cpp

```diff
@@ -156,6 +178,76 @@ void MadeEngine::resetAllTimers() {
 		_timers[i] = -1;
 }
 
+void MadeEngine::sayText(Common::String text, Common::TextToSpeechManager::Action action) const {
+	Common::replace(text, "_", "");
```

Comment:
Please add a comment here in form:
```C++
// Texts come in form of "_Here I say words_", so we remove the underscores
```
of course, use the real example, I do not know how those strings look like :D